telegram/_chatboost.py

```diff
@@ -0,0 +1,408 @@
+            the message could have been deleted already. May be 0 if the message isn't sent yet.
+        user (:class:`telegram.User`, optional): User that won the prize in the giveaway if any.
+        is_unclaimed (:obj:`bool`, optional): :obj:`True`, if the giveaway was completed, but
+            there was no user to win the prize.
+
+    Attributes:
+        source (:obj:`str`): Source of the boost. Always
+            :attr:`~telegram.ChatBoostSource.GIVEAWAY`.
+        giveaway_message_id (:obj:`int`): Identifier of a message in the chat with the giveaway;
+            the message could have been deleted already. May be 0 if the message isn't sent yet.
+        user (:class:`telegram.User`): Optional. User that won the prize in the giveaway if any.
+        is_unclaimed (:obj:`bool`): Optional. :obj:`True`, if the giveaway was completed, but
+            there was no user to win the prize.
+    """
+
+    def __init__(
+        self,
+        giveaway_message_id: int,
+        user: Optional[User] = None,
+        is_unclaimed: Optional[bool] = None,
+        *,
+        api_kwargs: Optional[JSONDict] = None,
+    ):
+        super().__init__(source=self.GIVEAWAY, api_kwargs=api_kwargs)
+
+        with self._unfrozen():
+            self.giveaway_message_id: int = giveaway_message_id
+            self.user: Optional[User] = user
+            self.is_unclaimed: Optional[bool] = is_unclaimed
+
+
+class ChatBoost(TelegramObject):
+    """
+    This object contains information about a chat boost.
+
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`boost_id`, :attr:`add_date`, :attr:`expiration_date`,
+    and :attr:`source` are equal.
+
+    .. versionadded:: NEXT.VERSION
+
+    Args:
+        boost_id (:obj:`str`): Unique identifier of the boost.
+        add_date (:obj:`int`): Point in time (Unix timestamp) when the chat was boosted.
+        expiration_date (:obj:`datetime.datetime`): Point in time (Unix timestamp) when the boost
+            will automatically expire, unless the booster's Telegram Premium subscription is
+            prolonged.
+        source (:class:`telegram.ChatBoostSource`): Source of the added boost.
+
+    Attributes:
+        boost_id (:obj:`str`): Unique identifier of the boost.
+        add_date (:obj:`int`): Point in time (Unix timestamp) when the chat was boosted.
+        expiration_date (:obj:`datetime.datetime`): Point in time (Unix timestamp) when the boost
+            will automatically expire, unless the booster's Telegram Premium subscription is
+            prolonged.
+        source (:class:`telegram.ChatBoostSource`): Source of the added boost.
+    """
+
+    __slots__ = ("boost_id", "add_date", "expiration_date", "source")
+
+    def __init__(
+        self,
+        boost_id: str,
+        add_date: int,
+        expiration_date: int,
```

Comment:
```suggestion
        add_date: datetime,
        expiration_date: datetime,
```
please also update the docstring of `add_date` accordingly. the "Unix timestamp" can be removed as well

Reply:
yep yep did that gonna push soon